ws/adapter.go

```diff
@@ -25,8 +25,8 @@ var (
 	// ErrFailedSubscription indicates that client couldn't subscribe to specified channel.
 	ErrFailedSubscription = errors.New("failed to subscribe to a channel")
 
-	// ErrFailedConnection indicates that service couldn't connect to message broker.
-	ErrFailedConnection = errors.New("failed to connect to message broker")
+	// ErrFailedConnect indicates that service couldn't connect to message broker.
```

Comment:
I prefer `FailedConnection`

Reply:
@drasko Should I use `ErrFailedDisconnection`?

Reply:
No need to change, this was just neat-picking, and probably not correct. Let's move on, I approved.